moveit_planners/chomp/chomp_motion_planner/include/chomp_motion_planner/chomp_optimizer.h

```diff
@@ -81,6 +81,7 @@ class ChompOptimizer
   {
     return is_collision_free_;
   }
+  typedef std::vector<Eigen::Vector3d, Eigen::aligned_allocator<Eigen::Vector3d>> StlVectorEigenVector3d;
```

Comment:
MoveIt uses the `eigen_stl_containers in many places already.
Please use their datastructures instead of redefining them here. (`EigenSTL::vector_Vector3d`)